apps/src/music/blockly/toolbox.js

```diff
@@ -196,3 +197,43 @@ export const baseToolbox = {
     }
   ]
 };
+
+const baseToolboxSimple = {
```

Comment:
Would we still want some other logic/control blocks? e.g. I can see a repeat block being useful when used in conjunction with Go to Next Measure

Reply:
Yes, that makes sense!  I was also starting to think about a repeat block.  I think I'll merge this first, and work on that soon.

Reply:
You called it! Works great: https://github.com/code-dot-org/code-dot-org/pull/49579